api/src/main/java/io/opencensus/trace/unsafe/CtxUtils.java

```diff
@@ -0,0 +1,46 @@
+package io.opencensus.trace.unsafe;
+
+import io.opencensus.trace.ContextManager;
+import io.opencensus.trace.Ctx;
+import io.opencensus.trace.Span;
+
+public class CtxUtils {
```

Comment:
I originally wanted to keep `ContextUtils`, but unfortunately its method signatures require working with gRPC context.

I've introduced the `Ctx` interface to allow using implementations not using gRPC, and had to declare this new class to keep the original `ContextUtils` untouched.